semgrep-core/src/analyzing/Dataflow_svalue.mli

```diff
@@ -1,5 +1,7 @@
 type mapping = AST_generic.svalue Dataflow_core.mapping
+type constness_type = Constant | NotAlwaysConstant [@@deriving show]
```

Comment:
Could reuse G.Cst? But might be nice to separate this out

Reply:
then maybe good to add a comment saying you know about G.Cst, but that you deliberately defined another one.